client/my-sites/plan-features-2023-grid/plans-2023-tooltip.tsx

```diff
@@ -12,7 +12,6 @@ const StyledTooltip = styled( Tooltip )`
 		border-radius: 4px;
 		min-height: 32px;
 		width: 210px;
-		display: flex;
```

Comment:
This is removed so that the <i> element won't take a whole block like so:
<img width="332" alt="image" src="https://user-images.githubusercontent.com/1842898/218403401-5a6ca044-e25d-496f-b6b4-225916ee3ce7.png">